### ISSUE TITLE
Test-VmwDscConfiguration doesn't work

### ISSUE THREAD

**Issue by tralalas** [vmware/dscr-for-vmware#323]
Hello,
I'm trying to test the VMware.vSphereDSC and VMware.PSDesiredStateConfiguration.
I have created the following config file as per the examples from https://github.com/vmware/dscr-for-vmware/blob/master/VMware.PSDesiredStateConfiguration.md

dsc_test01.ps1
Param(
    [Parameter(Mandatory = $true)]
    [ValidateNotNullOrEmpty()]
    [string]
    $Server,

    [Parameter(Mandatory = $true)]
    [ValidateNotNullOrEmpty()]
    [System.Management.Automation.PSCredential]
    $Credential
)

Configuration Datacenter_Config {
    
    Param(
        [string]
        $Server,

        [System.Management.Automation.PSCredential]
        $Credential
    )

    Import-DscResource -ModuleName VMware.vSphereDSC
    
    Node localhost {
        Datacenter MyDatacenter {
            Server = $Server
            Credential = $Credential
            Name = 'MyDatacenter'
            Location = ''
            Ensure = 'Present'
        }
    }
}

after that from a powershell with adminrights I'm running the following:
1) to provide the creds of the vcenter $Credential = Get-Credential
2)to create the new vmwdsc object with the parameters
$newVmwDscConfigParams = @{
    Path = '.\DSC_test01.ps1'
    Parameters = @{
        Server = 'myvcenterfqdn'
        Credential = $Credential
    }
}
3)$dscConfiguration = New-VmwDscConfiguration @newVmwDscConfigParams
4)trying to test Test-VmwDscConfiguration -Configuration $dscConfiguration
I get the following error:
**Select-Object : Property "InvokeResult" cannot be found.**
At C:\Program Files\WindowsPowerShell\Modules\VMware.PSDesiredStateConfiguration\1.0.0.16\Functions\Public\Test-VmwDscC
onfiguration.ps1:76 char:35
+ ...  = ($testResults | Select-Object -ExpandProperty InvokeResult | Where ...
+                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (System.Collections.Hashtable:PSObject) [Select-Object], PSArgumentExce
   ption
    + FullyQualifiedErrorId : ExpandPropertyNotFound,Microsoft.PowerShell.Commands.SelectObjectCommand

if I run the Start-VmwDscConfiguration -Configuration $dscConfiguration everything is fine MyDatacenter is created without any issues.

I have tried to run it from both ps 5.1 and 7.0 but in 7.0 I get an error that is not able to find the modules in C:\Program which I guess is due to the known error of spaces.

Any idea why I get that error ?

**Comment by SimeonGerginov** [comment]
I've tested it on both **`PowerShell 5.1 and 7.0`**. On **`PowerShell 7.0`** everything works as expected. On **`PowerShell 5.1`** if you call the **`Test-VmwDscConfiguration`** cmdlet with the **`Detailed`** parameter like this:

```powershell
Test-VmwDscConfiguration -Configuration $dscConfiguration -Detailed
```

Everything also works as expected. The problem only arises when the **`Detailed`** parameter is not passed on **`PowerShell 5.1`**.

Till we address this issue, you can either use **`PowerShell 7.0`** in a folder with no spaces or pass the  the **`Detailed`** parameter on **`PowerShell 5.1`**.

On  **`PowerShell 7.0`** you can use the **`$env:PSModulePath`** variable and add to it a folder where you have the modules like this: 

```powershell
$env:PSModulePath += ';<Path to folder with the VMware modules>'
```

**Comment by SimeonGerginov** [comment]
The bug was fixed with PR #324, so it should work as expected with the latest version of the module.

**Comment by tralalas** [comment]
Hi Simeon,
Ok thank you so I was missing the -detailed parameter.

Thank you,
Sorin
